build-system/tasks/helpers.js

```diff
@@ -717,14 +723,82 @@ function buildExaminer(options) {
  * @param {!Object} options
  */
 function buildWebWorker(options) {
-  const opts = Object.assign({}, options);
   return compileJs('./src/web-worker/', 'web-worker.js', './dist/', {
     toName: 'ww.max.js',
     minifiedName: 'ww.js',
     includePolyfills: true,
-    watch: opts.watch,
-    minify: opts.minify || argv.minify,
+    watch: options.watch,
+    minify: options.minify || argv.minify,
+  });
+}
+
+/**
+ * Build AMP experiments.js.
+ *
+ * @param {!Object} options
+ */
+function buildExperiments(options) {
+  return compileJs(
+    './build/experiments/',
+    'experiments.max.js',
+    './dist.tools/experiments/',
+    {
+      watch: false,
+      minify: options.minify || argv.minify,
+      includePolyfills: true,
+      minifiedName: 'experiments.js',
+    }
+  );
+}
+
+/**
+ * Build amp-login-done-${version}.js file.
+ *
+ * @param {string} version
+ * @param {!Object} options
+ */
+function buildLoginDone(version, options) {
+  const buildDir = `build/all/amp-access-${version}/`;
+  const builtName = `amp-login-done-${version}.max.js`;
+  const minifiedName = `amp-login-done-${version}.js`;
+  const latestName = 'amp-login-done-latest.js';
+  return compileJs('./' + buildDir, builtName, './dist/v0/', {
+    watch: false,
+    includePolyfills: true,
+    minify: options.minify || argv.minify,
+    minifiedName,
+    latestName,
+    extraGlobs: [
+      buildDir + 'amp-login-done-0.1.max.js',
+      buildDir + 'amp-login-done-dialog.js',
+    ],
+  });
+}
+
+/**
+ * Build amp-web-push publisher files HTML page.
+ *
+ * @param {!Object} options
+ */
+function buildWebPushPublisherFiles(options) {
```

Comment:
Move this back to `dist.js`, since it's only used in there.

Reply:
its either that or we should move `buildAlp`, `buildExaminer` and the like from helpers and back to `dist.js` too

Reply:
done.